### PR TITLE
improve karmada init help output 

### PR DESCRIPTION
**Pull request by my-git9**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #
improve karmada init help output

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Improve karmada init help output
```

### CODE REVIEW

File: pkg/karmadactl/cmdinit/cmdinit.go [hunk 1]
```diff
@@ -107,19 +107,19 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 	crdURL := fmt.Sprintf("https://github.com/karmada-io/karmada/releases/download/%s/crds.tar.gz", releaseVer.FirstMinorRelease())
 	flags.StringVar(&opts.CRDs, "crds", crdURL, "Karmada crds resource.(local file e.g. --crds /root/crds.tar.gz)")
 	flags.Int32VarP(&opts.KarmadaAPIServerNodePort, "port", "p", 32443, "Karmada apiserver service node port")
-	flags.StringVarP(&opts.KarmadaDataPath, "karmada-data", "d", "/etc/karmada", "karmada data path. kubeconfig cert and crds files")
+	flags.StringVarP(&opts.KarmadaDataPath, "karmada-data", "d", "/etc/karmada", "Karmada data path. kubeconfig cert and crds files")
 	flags.StringVarP(&opts.KarmadaAPIServerImage, "karmada-apiserver-image", "", "", "Kubernetes apiserver image")
-	flags.Int32VarP(&opts.KarmadaAPIServerReplicas, "karmada-apiserver-replicas", "", 1, "karmada apiserver replica set")
+	flags.Int32VarP(&opts.KarmadaAPIServerReplicas, "karmada-apiserver-replicas", "", 1, "Karmada apiserver replica set")
 	flags.StringVarP(&opts.KarmadaSchedulerImage, "karmada-scheduler-image", "", fmt.Sprintf("swr.ap-southeast-1.myhuaweicloud.com/karmada/karmada-scheduler:%s", releaseVer.PatchRelease()), "karmada scheduler image")
```
Comment (wuyingjun-lucky) [inline pkg/karmadactl/cmdinit/cmdinit.go:113]: `Karmada scheduler image`

File: pkg/karmadactl/cmdinit/cmdinit.go [hunk 2]
```diff
@@ -107,19 +107,19 @@ func NewCmdInit(parentCommand string) *cobra.Command {
...
-	flags.Int32VarP(&opts.KarmadaSchedulerReplicas, "karmada-scheduler-replicas", "", 1, "karmada scheduler replica set")
+	flags.Int32VarP(&opts.KarmadaSchedulerReplicas, "karmada-scheduler-replicas", "", 1, "Karmada scheduler replica set")
 	flags.StringVarP(&opts.KubeControllerManagerImage, "karmada-kube-controller-manager-image", "", "", "Kubernetes controller manager image")
-	flags.Int32VarP(&opts.KubeControllerManagerReplicas, "karmada-kube-controller-manager-replicas", "", 1, "karmada kube controller manager replica set")
+	flags.Int32VarP(&opts.KubeControllerManagerReplicas, "karmada-kube-controller-manager-replicas", "", 1, "Karmada kube controller manager replica set")
 	flags.StringVarP(&opts.KarmadaControllerManagerImage, "karmada-controller-manager-image", "", fmt.Sprintf("swr.ap-southeast-1.myhuaweicloud.com/karmada/karmada-controller-manager:%s", releaseVer.PatchRelease()), "karmada controller manager  image")
```
Comment (wuyingjun-lucky) [inline pkg/karmadactl/cmdinit/cmdinit.go:117]: `Karmada controller manager image`

**Comment by RainbowMango** [comment]
I'm so sorry that I missed this PR on my radar. 
@my-git9 Could you please solve the conflicts? After that, you can assign it to me.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**